crates/control/src/behavior/head.rs

```diff
@@ -14,10 +15,26 @@ impl<'cycle> LookAction<'cycle> {
     }
 
     pub fn execute(&self) -> HeadMotion {
-        HeadMotion::LookAt {
-            target: self.world_state.position_of_interest,
-            image_region_target: ImageRegion::Center,
-            camera: None,
+        if self.world_state.robot.role == Role::Keeper {
+            if let Some(ball_position) = self.world_state.ball {
+                HeadMotion::LookAt {
+                    target: ball_position.ball_in_ground,
+                    image_region_target: ImageRegion::Center,
+                    camera: None,
+                }
```

Comment:
We have to check whether this has negative consequences for the keeper localization... but okay for now

(resolve when read)